.gitlab-ci.yml

```diff
@@ -562,34 +573,35 @@ build-rustdoc:
   variables:                       &docker-build-vars
     <<:                            *default-vars
     GIT_STRATEGY:                  none
-    DOCKERFILE:                    $PRODUCT.Dockerfile
-    IMAGE_NAME:                    docker.io/parity/$PRODUCT
+    DOCKERFILE:                    "${PRODUCT}.Dockerfile"
+    IMAGE_NAME:                    "docker.io/${DOCKER_ORG}/${PRODUCT}"
```

Comment:
yaml anticipates strings in the values by default, no need to put them into quotation marks

Reply:
I have bash in mind when I see $VARIABLE :))